### PR TITLE
 message-list repositioning improved and some necessary css class added…

### PR DESCRIPTION
**Pull request by basafilm**
… to Message, MessageList, MessageForm and Channels components

# Description

To improve repositioning of message lists in chat needed to correct many wrong implemented css and create some new classes for some related components that have not css file. The related change implemented in Message, MessageList, MessageForm and Channels components.

Fixes # (issue)

# How to test?

Now If some one send message it scroll to bottom of the texts, fixed  only with css !

# Checklist

- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/MessageForm/SendMessageForm.js [hunk 1]
```diff
@@ -27,14 +27,19 @@ function SendMessageForm({ channelId, userId }) {
...
-          alt=""
-        />
-        <input
-          value={input}
-          onChange={(event) => setInput(event.target.value)}
-        />
+        <div className="new-message-form-imag">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
+            alt=""
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:33]: Always add a readable alt caption.

File: src/client/components/MessageForm/SendMessageForm.js [hunk 2]
```diff
@@ -27,14 +27,19 @@ function SendMessageForm({ channelId, userId }) {
...
-          src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
-          alt=""
-        />
-        <input
-          value={input}
-          onChange={(event) => setInput(event.target.value)}
-        />
+        <div className="new-message-form-imag">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/List-Icon.svg/768px-List-Icon.svg.png"
```
Comment (benna100) [inline src/client/components/MessageForm/SendMessageForm.js:32]: Why are you using this image? It looks nothing like the image in the mockup

File: src/client/components/MessageList/MessageList.js [hunk 3]
```diff
@@ -3,22 +3,24 @@
 import React from 'react';
 import Message from '../Message/Message';
 import PropTypes from 'prop-types';
-import '../Message/Message.css';
+import './MessageList.css';
```
Comment (benna100) [inline src/client/components/MessageList/MessageList.js:6]: Nice!! This is a great change. Shows that you have an eye for detail. That it was not the correct file and then fixing that

File: src/client/containers/Channel/Channel.js [hunk 4]
```diff
@@ -52,18 +53,18 @@ export default function Channel() {
   }
   if (messages.length === 0) {
     return (
-      <>
+      <div className="messageList-sendMessageForm-container">
```
Comment (benna100) [inline src/client/containers/Channel/Channel.js:56]:
```suggestion
      <div className="message-list-send-message-form-container">
```

**Review by benna100 — CHANGES_REQUESTED** [review]
You are using lots of wrappers and containers. Maybe you need them i dont know but the markup should always be as clean as possible. This is just general advice. Dont change anything here. Super cool that you fixed it using only css. Looking forward to trying it out. 

There are a fex small fixes. 

Does the sendMessage form look like in the mockup now? Also on mobile?

File: src/client/components/MessageForm/SendMessageForm.js (hunk 1 above)
Comment (basafilm) [reply src/client/components/MessageForm/SendMessageForm.js:33]: I worked only on CSS part not javascript for example adding alt. It was there , like that, but i will fix that also.

File: src/client/components/MessageForm/SendMessageForm.js (hunk 2 above)
Comment (basafilm) [reply src/client/components/MessageForm/SendMessageForm.js:32]: I did not add any image . It was there like that. but i fixed its flexible size to screen size.

Comment (basafilm) [reply src/client/components/MessageForm/SendMessageForm.js:32]: I agree . It should not be there . instead there should be file upload system and its Icone.

Comment (basafilm) [reply src/client/components/MessageForm/SendMessageForm.js:32]: Screenshot 2020-10-06 at 10 00 35

File: src/client/components/MessageList/MessageList.css [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
-  padding: 20px;
+.chat-message-container {
+  width: 100%;
+  height: 90%;
+  overflow: auto;
+  display: flex;
+  flex-direction: column-reverse;
+}
+
+.chat-message-ul {
```
Comment (tpetrina) [inline src/client/components/MessageList/MessageList.css:9]: Can I suggest dropping the `ul` part from the class name?

**Pull request merged**